int32 fiducial_id
string tf_id
WaypointType type